src/Data/List/Relation/Binary/Sublist/Setoid/Properties.agda

```diff
@@ -36,6 +37,15 @@ open SetoidEquality S using (_≋_; ≋-refl)
 open SetoidSublist S hiding (map)
 open SetoidMembership S using (_∈_)
 
+------------------------------------------------------------------------
+-- Least element
+
+module _ where
+
+  []⊆ : ∀ xs → [] ⊆ xs
+  []⊆ [] = []
+  []⊆ (x ∷ xs) = x ∷ʳ []⊆ xs
+
```

Comment:
Is this operation not [already exported/exportable](https://github.com/andreasabel/agda-stdlib/blob/merge-is-sublist/src/Data/List/Relation/Binary/Sublist/Heterogeneous.agda#L37-L39) from
`Data.List.Relation.Binary.Sublist.Heterogeneous` as `minimum`, so a simple renaming-on-import would suffice?

Reply:
Thanks!  I hadn't found the `minimum` function.